src/main/java/com/bulkbuy/enterprise/controller/NavigationController.java

```diff
@@ -48,6 +51,7 @@ public Order saveOrder(@RequestBody Order order) throws Exception {
         catch (Exception ex)
         {
             //TODO: LOGGING/Exception handling
+            log.error(ex.getMessage());
```

Comment:
In a catch block, make sure you have some way to log the exception

Reply:
Good.